databricks_cli/pipelines/cli.py

```diff
@@ -309,57 +469,52 @@ def _get_pipeline_url(api_client, pipeline_id):
     return urljoin(base_url, "#joblist/pipelines/{}".format(pipeline_id))
 
 
-def _write_spec(src, spec):
-    """
-    Writes the spec at src as JSON.
-    """
-    data = json.dumps(spec, indent=2) + '\n'
-    with open(src, 'w') as f:
-        f.write(data)
-
-
 def _validate_pipeline_id(pipeline_id):
     """
-    Checks if the pipeline ID is not empty and contains only hyphen (-),
-    underscore (_), and alphanumeric characters.
+    Checks if the pipeline ID is not empty.
     """
     if pipeline_id is None or len(pipeline_id) == 0:
         error_and_quit(u'Empty pipeline ID provided')
-    if not set(pipeline_id) <= PIPELINE_ID_PERMITTED_CHARACTERS:
-        message = u'Pipeline ID {} has invalid character(s)\n'.format(pipeline_id)
-        message += u'Valid characters are: _ - a-z A-Z 0-9'
-        error_and_quit(message)
```

Comment:
@pradeepgv-db @mukulmurthy Can pipeline IDs be whatever you want?

Reply:
When creating new pipelines users can't specify pipeline ID anymore, DLT creates pipeline ID.

Reply:
Thanks!